packages/react/src/components/UI/Container.tsx

```diff
@@ -9,8 +8,8 @@ const containerDefaultStyles = css({
   variants: {
     direction: {
       horizontal: {
-        flexDirection: 'row',
-        margin: '4px 0',
+        display: 'grid',
+        gridTemplateColumns: 'repeat(auto-fit, minmax(48px, 1fr))',
```

Comment:
maybe we should add this to the variables so anyone can change it easily?

Reply:
oh where?